alacritty/src/display.rs

```diff
@@ -471,7 +478,22 @@ impl Display {
 
             self.renderer.with_api(&config.ui_config, config.cursor, &size_info, |mut api| {
                 // Iterate over all non-empty cells in the grid.
-                for cell in grid_cells {
+                for mut cell in grid_cells {
+                    // Invert the active match in vi-less search.
+                    let cell_point = Point::new(cell.line, cell.column);
+                    if viewport_match
+                        .as_ref()
+                        .map_or(false, |viewport_match| viewport_match.contains(&cell_point))
+                        && !visible_selection
+                            .map_or(false, |selection| selection.contains(cell.column, cell.line))
```

Comment:
I thin it shouldn't work with expanded selection, like wide chars.

Reply:
You're probably right, yes.